torch/_tensor_docs.py

```diff
@@ -4262,6 +4349,11 @@ def callable(a, b) -> number
 are like normal tensors, but they carry no data.
 """)
 
+add_docstr_all('is_sparse',
+               r"""
+Is ``True`` if the Tensor uses sparse storage layout, ``False`` otherwise.
```

Comment:
"Returns True if self is sparse, False otherwise."

Reply:
Note that `is_sparse` is an attribute, not a method, and the current style corresponds to documentations of many other `is_*` attributes (see `is_cuda`, `is_quantized`, `is_meta`). So, I would ignore this suggestion unless you think the docs of `is_*` attributes need to be updated similarly.

Reply:
Aha, good point. My mistake.